README.md

```diff
@@ -99,6 +99,16 @@ ctest -j4
 ```
 will run the tests with some parallelism.
 
+You can also add `testeth` options using
+````
+ctest -j4 -DTESTETH_ARGS="--verbosity 5"
```

Comment:
note tha boost options are not applied here. you can not select test suite or test case. but you could set --singletest --singlenet and so on

Reply:
Ohh, I want to add a few words about that, then.